xAPI.md

```diff
@@ -507,6 +507,41 @@ __Verb__: Defines the action being done by the Actor within the Activity within
+
+##### Activity Data and Metadata
+
+![xAPI Activity Data and Metadata](/xAPIMetadata.jpg)
+
+Understanding how a single activity (as uniquely identified by its IRI) is produced, populated, updated, and 
+consumed is a key concept in xAPI. Figure 2 shows this process.  xAPI Statements with the Activity objects have  
+metadata properties that may be populated.  This is done through the  Activity Definition Object.  Within the 
+Activity is an IRI which also could have metadata located at where it resolves to. (Note that this is different  
+from the moreinfo  URL in the Activity Definition, which could also have information located at it, but is not 
+involved in any way in this information flow.) The base portion of the IRI as well as any metadata where the IRI resolves is under the control of the Metadata Provider.  Any metadata at the IRI is the authoritative source of metadata, and could be used to populate the LRS's aggregated version of the Activity's metadata (Activity Definition) instead of what it receives from Statements. A Metadata Consumer can access Metadata directly from the authority or can query the Activities Resource.  
+
+
+##### Agent and Profile Management
+
+![xAPI Activity Data and Metadata](/xAPIAgentsAndEndpoints.jpg)
+
+xAPI grants the framework for allowing selective access to one's personal data.  This is done through the management of what we've come to call personas.  In xAPI, each persona represents the "I" in "I did this" and is 
+logically the subject of the Statement.  A user sending records to an LRS could have multiple personas associated with him or her.  In Figure 3, a single user can access multiple Services(or apps, etc.), each of which could have a different persona.  More likely is that personas exist with a common purpose, such as work, home, school, social, etc.  Each of these Services send data to the LRS. Within the LRS now, there are Statements from three different 
+personas of the same user.  The LRS can aggregate all of the information of each of the personas into one "Person"  
+Object and send it through the Agents Resource.  
+
+Data kept about each persona is not restricted to Statements.  Data can also be kept in an Agent Profile, which 
+use the same personas as defined in the Statements.  An Agent Profile could be used to see learner preferences or 
```

Comment:
As my comment on the diagrams, I think this would be clearer without reference to Agent Profiles.